styx-cli/src/main/java/com/spotify/styx/cli/Main.java

```diff
@@ -484,22 +489,26 @@ private String apiUrl(CharSequence... parts) {
   }
 
   private ByteString send(Request request) throws ExecutionException, InterruptedException {
-    return sendAsync(request).toCompletableFuture().get();
+    // TODO: Move requireSuccess use into callers of send()
```

Comment:
The `send` method already threw `RuntimeException` on 404 so this just restores that behavior. I think that might not belong in `send` though hence the TODO.

https://github.com/spotify/styx/pull/130/files#diff-0a3c00857227c1cef7cc1e40e3d912acL454